__snapshots__/Medium.md

```diff
@@ -0,0 +1,7 @@
+# `Medium`
```

Comment:
These elements moving to `<slot></slot>` only shadow DOM, I wonder if we'd be better off not having the snapshot tests for them?